Workade/Views&ViewModels/Magazine/TipDetail/TipDetailViewController.swift

```diff
@@ -8,25 +8,79 @@
+        collectionView.backgroundColor = .clear
+        collectionView.register(TipDetailCell.self, forCellWithReuseIdentifier: TipDetailCell.identifier)
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
         
-        return label
+        return collectionView
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        tipDetailCollectionView.dataSource = self
+        tipDetailCollectionView.delegate = self
+        
         setupLayout()
     }
     
     func setupLayout() {
-        view.addSubview(sampleLabel)
+        view.addSubview(tipDetailCollectionView)
         NSLayoutConstraint.activate([
-            sampleLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            sampleLabel.topAnchor.constraint(equalTo: view.topAnchor)
+            tipDetailCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            tipDetailCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            tipDetailCollectionView.topAnchor.constraint(equalTo: view.topAnchor, constant: 24),
+            tipDetailCollectionView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
         ])
     }
 }
+
+extension TipDetailViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let width = (UIScreen.main.bounds.width - 60) / 2
+        
+        return CGSize(width: width, height: width * 1.3)
+        }
+}
+
+extension TipDetailViewController: UICollectionViewDelegate {
+    
+}
+
+// TODO: 추후 요청으로 처리
+extension TipDetailViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return titleArray.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: TipDetailCell.identifier, for: indexPath) as? TipDetailCell else {
+            return UICollectionViewCell()
+        }
+        cell.titleLabel.text = titleArray[indexPath.row]
+        cell.bookMarkButton.addTarget(self, action: #selector(clickedBookmarkButton(sender:)), for: .touchUpInside)
+        
+        return cell
+    }
+    
+    // TODO: 추후 데이터 로직과 연결하여 수정
+    @objc
+    func clickedBookmarkButton(sender: UIButton) {
+        if sender.currentImage
+            == UIImage(systemName: "bookmark") {
+            sender.setImage(UIImage(systemName: "bookmark.fill"), for: .normal)
+        } else {
+            sender.setImage(UIImage(systemName: "bookmark"), for: .normal)
+        }
+    }
```

Comment:
```suggestion
    @objc
    func clickedBookmarkButton(sender: UIButton) {
        var image: UIImage?
        let config = UIImage.SymbolConfiguration(pointSize: 17, weight: .semibold)
        if sender.currentImage == UIImage(systemName: "bookmark", withConfiguration: config) {
            image = UIImage(systemName: "bookmark.fill", withConfiguration: config)
            sender.setImage(image, for: .normal)
        } else {
            image = UIImage(systemName: "bookmark", withConfiguration: config)
            sender.setImage(image, for: .normal)
        }
    }
```

[P2] 북마크 버튼 굵기를 반영하는 코드를 추가했습니다!! 따로 SF 이미지에 대한 폰트 가이드라인이 없어서 설명이 부족했던 것 같아 죄송스럽습니다..

Reply:
[P5] 추가로 저는 @objc 또한 스유에서 @State, @Binding 등 @ 키워드를 이어서 쓰던 습관이 있어서 한줄로 붙여쓰곤 하는데 @objc을 따로 줄바꿈을 하신 이유가 있으신가요?? 이점이 있다면 알려주시면 감사하겠습니다!

Reply:
[P2] 반영하였습니다.

[P5] 별 다른 이점없이 Lint 오류가 발생하여 개행을 넣었습니다. SwiftLint에서 잡더라구요 👍